### PR TITLE
add search for player page

### PR DESCRIPTION
**Pull request by kristianeaw**

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
+import { FPLBOT_APP_URL } from "../utils/envconfig";
 
 export default function Index() {
   return (
     <div>
       <Header />
       <Features />
       <AddToSlackForm />
+      <a
+        href={`${FPLBOT_APP_URL}/search`}
```
Comment (skjelbek) [inline pages/index.tsx:16]: Could simply be `/search` perhaps (just like `/search.png` on line 28). Clicking the link in the PR environment took me to a 404 in prod, so making it relative instead would solve that, I guess.

File: pages/search.tsx [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+              </td>
+              <td className="text-left md:text-center border-grey-light border hover:bg-gray-100 p-3 text-red-400 hover:text-red-600 hover:font-medium cursor-pointer">
+                <a
+                  href={`https://fantasy.premierleague.com/entry/${data.entry}/history`}
+                  className="block"
+                  target="_blank"
+                >
+                  ➡️
+                </a>
+              </td>
```
Comment (skjelbek) [inline pages/search.tsx:164]: I've added a `verifiedEntryEmoji` property now, which currently just holds a chess pawn for Carlsen. Could maybe fit that in somewhere? 😉

File: services/search.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+export interface FplApiResponse {
+  hits: PlayerEntry[];
+}
+
+export interface PlayerEntry {
+  id: number;
+  entry: number;
+  realName: string;
+  teamName: string;
```
Comment (skjelbek) [inline services/search.ts:11]: + `verifiedEntryEmoji`

**Comment by skjelbek** [comment]
🤩 👏

**Review by skjelbek — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (kristianeaw) [reply pages/index.tsx:16]: 🤦‍♀️ Jeg er dum

Comment (johnkors) [reply pages/index.tsx:16]: Så vi kan egentlig fjerne `FPLBOT_APP_URL` helt, og bare bruke relative urls..?

Comment (kristianeaw) [reply pages/index.tsx:16]: burde gå bra det ja, bare hjernen min som ikke fungerte når jeg implementerte det 😄

**Pull request merged**